pkg/build/builder/source.go

```diff
@@ -27,6 +26,10 @@ const (
 	urlCheckTimeout = 16 * time.Second
 )
 
+var (
+	GitAuthenticationError, GitNotFoundError error
```

Comment:
These should be error types. See for example https://github.com/openshift/origin/blob/3cf7f3cccd5b7b95448568ba8e2b3c81cc6ab9af/pkg/deploy/controller/deploymentconfig/controller.go#L45